presidio-api/cmd/presidio-api/docs/swagger.yaml

```diff
@@ -0,0 +1,1890 @@
+        x-go-name: StreamsTemplateId
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  Transformation:
+    description: Transformation represents the transformation types - how the sensitive
+      text will change
+    properties:
+      fPEValue:
+        $ref: '#/definitions/FPEValue'
+      hashValue:
+        $ref: '#/definitions/HashValue'
+      maskValue:
+        $ref: '#/definitions/MaskValue'
+      redactValue:
+        $ref: '#/definitions/RedactValue'
+      replaceValue:
+        $ref: '#/definitions/ReplaceValue'
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+  Trigger:
+    description: Trigger defines the trigger for a new job to start
+    properties:
+      schedule:
+        $ref: '#/definitions/Schedule'
+    type: object
+    x-go-package: github.com/microsoft/presidio/presidio-api/cmd/presidio-api/docs/vendor/github.com/Microsoft/presidio-genproto/golang
+info:
+  description: Documentation for Presidio API Swagger.
+  title: Presidio.
+  version: "1.0"
+paths:
+  /analyzer/recognizers/:
+    get:
+      description: Get all recognizers
+      operationId: getAllRecognizersRequest
+      responses:
+        "200":
+          $ref: '#/responses/allRecognizersGetResponse'
+      tags:
+      - analyze
+  /analyzer/recognizers/{id}:
+    get:
+      description: Get an existing recognizer
+      operationId: getRecognizerRequest
+      parameters:
+      - description: Recognizer name
+        in: path
+        name: id
+        required: true
+        type: string
+        x-go-name: Id
+      responses:
+        "200":
+          $ref: '#/responses/recognizersGetResponse'
+      tags:
+      - analyze
+    post:
+      description: Insert a new recognizer
+      operationId: insertRecognizerRequest
+      parameters:
+      - description: Recognizer name
+        in: path
+        name: id
+        required: true
+        type: string
+        x-go-name: Id
+      - description: Recognizer properties
+        in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/RecognizerInsertOrUpdateRequest'
+      responses:
+        "200":
+          $ref: '#/responses/insertRecognizersResponse'
+      tags:
+      - analyze
+  /fieldTypes:
+    get:
+      description: Get all available field types
+      operationId: getFieldTypes
+      responses:
+        "200":
+          $ref: '#/responses/fieldTypesGetResponse'
+      tags:
+      - fields
+  /projects/{projectId}/analyze:
+    post:
+      description: Analyze text
+      operationId: analyzeRequest
+      parameters:
+      - description: The request body
+        in: body
+        name: Body
+        schema:
+          $ref: '#/definitions/AnalyzeRequest'
+      - format: int64
+        in: path
+        name: projectId
```

Comment:
What is the projectId for?

Reply:
It's a parameter you can use to have different configurations (templates and services) in place. @rabee333 worth adding to the swagger documentation.

Reply:
Added descriptions to ProjectID parameters